src/test/regress/output/multi_insert_select_behavioral_analytics_create_table.source

```diff
@@ -62,3 +62,370 @@ CREATE INDEX is_index3 ON users_table(value_1);
 CREATE INDEX is_index4 ON events_table(event_type);
 CREATE INDEX is_index5 ON users_table(value_2);
 CREATE INDEX is_index6 ON events_table(value_2);
+-- create a helper function to create types/functions on each node
+CREATE FUNCTION run_command_on_master_and_workers(p_sql text)
```

Comment:
I don't want to create such dependencies among the tests. 

I think in one day we should add this function to `citus_tools`. Also, if we one day decide to improve the regression tests such that they all share some common tests which are responsible for creating tables/types/functions etc., I'd move this function.